### PR TITLE
Done Changes

### PR DESCRIPTION
**Pull request by NishantGautam023**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/Nis/Nis.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+name: Nishant Gautam 
+institution:  Visvesvaraya Technological University(VTU)
+profile_pic: Nis.png
+quote: Make it or break it , curse it or rejoice it.The life of a developer is a never ending learning Path.
+github_user: NishantGautam023
+---
+
```
Comment (Mohammed-Shoaib) [inline _data/Nis/Nis.md:8]: Can you please remove the extra new line?

File: _data/Nis/Nis.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Nishant Gautam 
+institution:  Visvesvaraya Technological University(VTU)
+profile_pic: Nis.png
+quote: Make it or break it , curse it or rejoice it.The life of a developer is a never ending learning Path.
```
Comment (Mohammed-Shoaib) [inline _data/Nis/Nis.md:5]: Quote must **not** be longer than 100 characters, yours is **101** characters.

Comment (NishantGautam023) [reply _data/Nis/Nis.md]: Done can you check it now

**Comment by Mohammed-Shoaib** [comment]
Hey! Don't close the pull request, otherwise, the maintainers cannot merge your changes.

Kindly re-open it!

**Comment by NishantGautam023** [comment]
Kindly check that now .. Sorry

**Comment by Mohammed-Shoaib** [comment]
I am sorry I cannot open the pull request. You are the creator, you'll have to re-open it!

**Comment by NishantGautam023** [comment]
It's #3041 pull request

**Comment by Mohammed-Shoaib** [comment]
Please do not close it this time and wait for the maintainers to merge it.

**Pull request closed without merging**